package.json

```diff
@@ -17,8 +17,8 @@
     "clean": "lerna clean -y && rm -Rf node_modules",
     "check": "lerna run check --since HEAD --exclude-dependents",
     "check-ci": "lerna run check",
-    "test": "lerna run test",
-    "test-ci": "lerna run test-ci",
+    "test": "rimraf .nyc_output && lerna exec -- nyc --no-clean --cwd ../.. --reporter=none npm run test && npm run report-coverage",
```

Comment:
do we want to have coverage always or as a separate job like `test-cov`? won't it take more to run tests with coverage? or is intentional?

Reply:
It'll take a bit longer, yes... but `npm test` for the full repo already takes at least 10 minutes, so it's not like it's something that one would run on a regular basis during development (I only do it before submitting a PR, and admittedly only sometimes ... CI is there to do that work for us)

Reply:
Oh i see, the coverage runs only on the root package .. didn't notice.

Reply:
Yeah, I was referring to `test` and `test-ci` of the root package :)